srcpkgs/gtk+4/template

```diff
@@ -0,0 +1,74 @@
+# Template file for 'gtk+4'
+pkgname=gtk+4
+version=3.94.0
+revision=1
+wrksrc="gtk+-${version}"
+build_style=meson
+build_helper="gir"
+configure_args="-Dman-pages=true
+ -Dbuild-tests=false
+ -Dmedia=all
```

Comment:
This links against both ffmpeg and gstreamer, not quite so sure if we want that.